src/containers/shared/utils.js

```diff
@@ -152,6 +152,9 @@ export const getLocalizedCurrencySymbol = (lang = 'en-US', currency = 'USD') =>
 };
 
 export const formatLargeNumber = (d = 0, digits = 4) => {
+  let variableDigits = 4;
```

Comment:
Why does this start at 4?

Reply:
The can be very very small. So displaying a minimum of 4 digits might give us a sense. If we try to reduce that, most small numbers won't have significant information to display.

Reply:
Why does it not start at `digits`?